changelog/13126.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:deprecation
+auth/kubernetes: `disable_iss_validation` and `issuer` configuration fields deprecation  [#127](https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/127)
+```
```

Comment:
I'm thinking this should have a `change` entry too, in the same file, something like:

````
```release-note:change
auth/kubernetes: default `disable_iss_validation` to true [[GH-127](https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/127)]
``` 
````